internal/cmd/functional-benchmarker/runners/cluster/init.go

```diff
@@ -0,0 +1,9 @@
+package cluster
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+)
+
+var (
+	log = utils.InitLogger("cluster")
```

Comment:
`InitStaticLogger()`?

Reply:
same here as this becomes a package level logger..